symplyphysics/laws/thermodynamics/energy_to_melt_from_mass.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+from symplyphysics.laws.thermodynamics import energy_from_combustion as combustion_energy_law
+
+# Description
+## The heat of melting is the amount of heat that must be brought to a solid
+# crystalline substance at constant pressure in order to completely transfer it
+# to a liquid state.
+## Q = k_lambda * m
+## Where:
+## Q = energy, required to melt solid matter
+## k_lambda - specific heat of melting
+## m - mass of solid matter
+
+amount_energy = Symbol("amount_energy", units.energy)
+specific_heat_melting = Symbol("specific_heat_melting", units.energy / units.mass)
+mass_of_matter = Symbol("mass_of_matter", units.mass)
+
+# Important note! Although we substitute the specific heat of melting
+# into the specific heat of combustion in the formula of the thermal energy
+# released during the combustion of a substance, these values should be distinguished,
+# they are not equivalent!
+energy_from_combustion_value = combustion_energy_law.law.subs({
```

Comment:
Don't do it. Write in the description about similarities. Also mention specific heat difference near the specific heat usage.

Reply:
Like in commit 07c9bb6 ?

Reply:
Don't reuse another law.